docs/release-notes/NuGet-7.0.md

```diff
@@ -1,26 +1,296 @@
 ---
 title: NuGet 7.0 Release Notes
 description: Release notes for NuGet 7.0 including new features, bug fixes, and DCRs.
-author: zivkan
+author: donnie-msft
+ms.date: 10/29/2025
 ms.topic: conceptual
 ---
-# NuGet 7.0 Release Notes
 
-<!--
-    This is intentionally not yet added to TOC.md.
-    When this version is ready for release:
-    1. Rename this file to the next version
-    2. Change the version strings in this file to the next version
-    3. Use the release tool to create the real release notes for the version ready for release
-    4. Add the release notes to TOC.yml
--->
+# NuGet 7.0 Release Notes
 
 NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**7.0**](https://nuget.org/downloads) | TBD | TBD |
+| [**7.0.0**](https://nuget.org/downloads) | [Visual Studio 2026 version 18.0.0](https://visualstudio.microsoft.com/downloads/) | [10.0.100](https://dotnet.microsoft.com/download/dotnet/10.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2026 with any .NET workload
+
+
+## Summary: What's New in 7.0.0
+
+* Details pane reflects Vulnerability Data from Audit Sources - [#14554](https://github.com/NuGet/Home/issues/14554)
+
+* When adding an http source in the options dialog, have the user click on a checkbox to more explicitly agree to AllowInsecureConnections being added - [#14377](https://github.com/NuGet/Home/issues/14377)
+
+* Error needed in Package Sources settings for HTTP source without AllowInsecureConnections - [#14367](https://github.com/NuGet/Home/issues/14367)
+
+* Implement programmatic validation API for Unified Settings - [#14359](https://github.com/NuGet/Home/issues/14359)
+
+* Use Regex UI validation on NuGet options pages - [#14358](https://github.com/NuGet/Home/issues/14358)
+
+* Create Unified Settings page for &quot;Package Source Mapping&quot; - [#14234](https://github.com/NuGet/Home/issues/14234)
```

Comment:
Does unified settings mean somerthing to our customers?
Should call it something else? Same applies. There are big enough to stay here, but wonder if there's a more impactful description.